src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -28,7 +28,10 @@ import {
   getApplicationDetails,
   verifyAllDocuments,
 } from "../../../services/applicationService";
-import { updateApplicationStatus } from "../../../services/benefits";
+import {
+  calculateBenefitAmount,
```

ESLint

[error] 'calculateBenefitAmount' is defined but never used.

(@typescript-eslint/no-unused-vars)